respect-lib-shared/src/commonMain/kotlin/world/respect/shared/viewmodel/manageuser/getstarted/GetStartedViewModel.kt

```diff
@@ -0,0 +1,126 @@
+package world.respect.shared.viewmodel.manageuser.getstarted
+
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import org.jetbrains.compose.resources.getString
+import world.respect.credentials.passkey.GetCredentialUseCase
+import world.respect.shared.generated.resources.Res
+import world.respect.shared.generated.resources.invalid_school_name
+import world.respect.shared.generated.resources.lets_get_started
+import world.respect.shared.navigation.JoinClazzWithCode
+import world.respect.shared.navigation.LoginScreen
+import world.respect.shared.navigation.NavCommand
+import world.respect.shared.navigation.RespectAppLauncher
+import world.respect.shared.resources.StringResourceUiText
+import world.respect.shared.viewmodel.RespectViewModel
+
+
+class GetStartedViewModel(
+    savedStateHandle: SavedStateHandle,
+    getCredentialUseCase: GetCredentialUseCase
+) : RespectViewModel(savedStateHandle) {
+
+    private val _uiState = MutableStateFlow(GetStartedUiState())
+    val uiState = _uiState.asStateFlow()
+
+    init {
+        viewModelScope.launch {
+            _appUiState.update { prev ->
+                prev.copy(
+                    title = getString(Res.string.lets_get_started),
+                    hideBottomNavigation = true,
+                    userAccountIconVisible = false,
+                    showBackButton = false
+                )
+            }
+        }
+        viewModelScope.launch {
+
+            viewModelScope.launch {
+                try {
+                    when (val credentialResult = getCredentialUseCase()) {
+                        is GetCredentialUseCase.PasskeyCredentialResult -> {
+                            viewModelScope.launch {
```

Comment:
.launch is not needed to call tryEmit

Reply:
Still an issue: to be fixed in all viewmodels

Reply:
The viewmodelscope.launch has been removed, however, the indentation has not been updated accordingly. That needs resolved ASAP.